src/components/interfaces/MOBILE_API.xml

```diff
@@ -552,6 +552,24 @@
         <element name="BOTH" />
     </enum>
 
+    <struct name="CloudAppProperties" since="5.1">
+        <param name="appName" type="String" maxlength="100" mandatory="true"></param>
+        <param name="appID" type="String" maxlength="100" mandatory="true"></param>
+        <param name="enabled" type="Boolean" mandatory="false">
+            <description>If true, cloud app will be included in HMI RPC UpdateAppList</description>
+        </param>
+        <param name="authToken" type="String" maxlength="65535" mandatory="false">
+            <description>Used to authenticate websocket connection on app activation</description>
+        </param>
+        <param name="cloudTransportType" type="String" maxlength="100" mandatory="false">
+            <description>Specifies the connection type Core should use</description>
+        </param>
+        <param name="hybridAppPreference" type="HybridAppPreference" mandatory="false">
+            <description>Specifies the user preference to use the cloud app version or mobile app version when both are available</description>
+        </param>
+        <param name="endpoint" type="String" maxlength="65535" mandatory="false"></param>
```

Comment:
You should be able to set this field or we need to include a note that this cannot be sent in a `Set` request